neurow/test/neurow/internal_api/endpoint_test.exs

```diff
@@ -31,6 +31,19 @@ defmodule Neurow.InternalApi.EndpointTest do
     assert call.status == 404
   end
 
+  test "other routes requires a JWT sent in x-interservice-authorization header" do
```

Comment:
@achouippe this ensure we can process both headers, once migration is done, we will fallback to only one key and remove the need for the array if thats ok for you.